src/components/Dashboard/TodoList.vue

```diff
@@ -384,30 +363,6 @@
                 <template v-else-if="isStatusPaid(item)">
                   <!-- no action button in this case -->
                 </template>
-
-                <!-- new annual report -->
-                <template v-else-if="isStatusNew(item) && isTypeAnnualReport(item)">
-                  <v-btn
-                    class="btn-file-now"
-                    color="primary"
-                    :disabled="isFileAnnualReportDisabled(item, index)"
-                    @click.stop="doFileNow(item)"
-                  >
-                    <span>File Annual Report</span>
-                  </v-btn>
-                </template>
-
-                <!-- new conversion (staff only) -->
-                <template v-else-if="isStatusNew(item) && isTypeConversion(item) && isRoleStaff">
-                  <v-btn
-                    class="btn-file-now"
-                    color="primary"
-                    :disabled="!item.enabled"
-                    @click.stop="doFileNow(item)"
-                  >
-                    <span>File Record Conversion</span>
-                  </v-btn>
-                </template>
```

Comment:
As mentioned above, this section was simply moved to the top of this if-then-else construct.